max_frames_to_skip: 15
dist_threshold: 150.0
center_threshold: 80.0
area_threshold: 3.0
body_ratio: 0.5
dt: 0.02
use_dim: true
use_vel: false
Q: [9.0, 9.0, 200.0, 200.0, 4.0, 4.0]
R: [2.0, 2.0, 200.0, 200.0, 2.0, 2.0]
min_bbox_width: 60
max_bbox_width: 400
min_bbox_height: 60
max_bbox_height: 400